### ISSUE TITLE
Add auto collection

### ISSUE THREAD

**Issue by just4omp** [kajdaq/stunning-chainsaw#18]
It will be very good to add auto register items in collection before absorb. In todays update added weekly collection.

**Comment by ragganxy** [comment]
I have a version that supports it. Interested?

**Comment by just4omp** [comment]
> I have a version that supports it. Interested?

Yes)

**Comment by ragganxy** [comment]
https://gofile.io/?c=61i1kN.  Let me know if it works for you!

**Comment by just4omp** [comment]
> https://gofile.io/?c=61i1kN. Let me know if it works for you!

https://ankulua.boards.net/thread/1692/black-desert-mobile-bdm-afk 
is it your topic?

**Comment by ragganxy** [comment]
no

**Comment by just4omp** [comment]
> no

ok. i think i will test it tomorrow. mb you can give some contact if this issue will be closed?

**Comment by ragganxy** [comment]
ragganxy@protonmail.com

**Comment by just4omp** [comment]
> [ragganxy@protonmail.com](mailto:ragganxy@protonmail.com)

btw does your script has some random time in repeatable actions?

**Comment by simhol** [comment]
yes

**Comment by just4omp** [comment]
> https://gofile.io/?c=61i1kN. Let me know if it works for you!

testing it for about 1h looks like everything works good.
Just mentioned one moment. Script collect spoils of war. Is it possible to make this function like on/off setting?

**Comment by maskman26** [comment]
> https://gofile.io/?c=61i1kN. Let me know if it works for you!

I've tried it but it seem that it is having an issue with returning home to sell junk. Please look at the log attached.
[log.txt](https://github.com/kajdaq/stunning-chainsaw/files/4532862/log.txt)

**Comment by ragganxy** [comment]
Looks like something went wrong after or during fusing crystals. Either it was stuck in the crystals menu or something unexpected was opened afterwards.. Are you using NOX and resolution 1280 x 720?

**Comment by ragganxy** [comment]
> > https://gofile.io/?c=61i1kN. Let me know if it works for you!
> 
> testing it for about 1h looks like everything works good.
> Just mentioned one moment. Script collect spoils of war. Is it possible to make this function like on/off setting?

Spoils isn't activated..that's strange!

**Comment by maskman26** [comment]
> Looks like something went wrong after or during fusing crystals. Either it was stuck in the crystals menu or something unexpected was opened afterwards.. Are you using NOX and resolution 1280 x 720?

Yes I am currently using NOX. 1280x720, settings in game is High and max texture.

**Comment by ragganxy** [comment]
Does this happen every time you start the script? Can you make a recording of it?

**Comment by maskman26** [comment]
> Does this happen every time you start the script? Can you make a recording of it?

Nope. Actually it worked, but does not repeat even after few hours.
